bemoji

```diff
@@ -229,9 +229,37 @@ add_to_recent() {
 # Set default clipboard util
 _clipper() {
     if [ -n "$BEMOJI_CLIP_CMD" ]; then
-        # shellcheck disable=SC2068
-        ${BEMOJI_CLIP_CMD[@]}
-    elif [ -n "$WAYLAND_DISPLAY" ] && command -v wl-copy >/dev/null 2>&1; then
+        read -r -a cmd_array <<< "$BEMOJI_CLIP_CMD"
+        
+        if [ "${#cmd_array[@]}" -eq 1 ]; then
+            case "${cmd_array[0]}" in
+                wl-copy)
+                    if command -v wl-copy >/dev/null 2>&1; then
+                        wl-copy
+                        return
```

Comment:
Since each branch of this conditional returns on its own, we may as well just return at the end of the conditional below instead of having each branch have its own return.